### DISCUSSION TITLE
Cash out and lay bet options in API

### DISCUSSION THREAD

**Discussion by placejam** [cloudbet/docs#57] [General]
Hi,

Going through some of the old documents, I found that some time back hadn't had lay option and cash out option available through APIs.

Could you please let me know if that has been enabled ?

Thanks

**Comment by kgravenreuth** [comment]
Hi @placejam 

Thanks for raising this issue again. As mentioned in #23 cash out and lay bet options are on our radar, but haven't been added yet. We will let you know when we have an update.
